.github/workflows/performance-tests-job-configs/config_AvroIOIT.txt

```diff
@@ -0,0 +1,30 @@
+###############################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+"--runner=DataflowRunner",
+"--project=apache-beam-testing",
+"--tempRoot=gs://temp-storage-for-perf-tests",
+"--bigQueryDataset=beam_performance",
+"--bigQueryTable=avroioit_results",
+"--influxMeasurement=avroioit_results",
+"--numberOfRecords=225000000",
+"--expectedHash=2f9f5ca33ea464b25109c0297eb6aecb",
+"--datasetSize=1089730000",
+"--numWorkers=5",
+"--autoscalingAlgorithm=NONE",
+"--influxDatabase=beam_test_metrics",
```

Comment:
same as https://github.com/apache/beam/pull/28450#discussion_r1333171697 a load test builder step to set common options (e.g., currently if influxHost has changed then it needs to change all files)

Reply:
@Abacn I added the `test-arguments-action` action in order to encapsulate common test arguments into one place and get rid of different config file templates (quotes, commas, etc.). Unfortunately, there is no a ton of common arguments for both Performance and Load Tests, and moreover we have a different set of arguments with different case which depends on the language (java, python and go)

Reply:
thanks! this is a good start. Other than the synthetic options, other options should share at least within a group and one can use action parameters. We can clean up as followup.